lib/livebook_cli/deploy.ex

```diff
@@ -10,12 +10,12 @@ defmodule LivebookCLI.Deploy do
   @impl true
   def usage() do
     """
-    Usage: livebook deploy [options] filename|directory
+    Usage: livebook deploy [options] filename|glob pattern
```

Comment:
It looks like two separate arguments. Let's do either

```suggestion
    Usage: livebook deploy [options] filename|glob
```

or

```suggestion
    Usage: livebook deploy [options] filename|glob-pattern
```